### PR TITLE
Bug Fixes for Power BI service UI changes.

### LINKED ISSUE #155: 08-perform-data-analysis-in-power-bi-desktop.md Exercise 1 lab step changes
#### **Priority**
Critical

#### **Lab/Demo Number/Name**
08-perform-data-analysis-in-power-bi-desktop.md
Exercise 1, Task 2, Step 2 and 3
and
Exercise 1 Task 3, Step 4 and 5

#### **Describe the bug**
Exercise 1, Task 2, Step 2 and 3

Step 2 states: "Select New > Dataset. This opens the Get Data page, and under Create new content > Files, select Get."

The interface has changed I believe. It should say: " Click upload, then Browse. 

Step 3 should now state: "Navigate to D:\PL300\Labs\08-perform-data-analysis-in-power-bi-desktop\Starter folder."

Exercise 1 Task 3, Step 4 and 5

Step 4 states: "Next, we repeat the steps to Get Data from lab 1. From the Home ribbon tab > Data group, select Power BI Datasets."

Firstly, I don't know what the phrase "we repeat the steps to Get Data from lab 1" means and should be deleted. 

Also, it seems the interface has changed. Step 4 just needs to state now: "On the Home tab, in the Data group, click Get Data, then Power BI datasets."

Step 5 should state: "On the Data hub dialog box, Select the Sales Analysis dataset, then click Connect." 

#### **Screenshots**
Task 2 step 2:

Lab 8-1 Step 2

Task 2 step 3:

Lab 8-1 Step 3

Task 3 Step 4:

Lab 8-2 Step 4

Task 3 Step 5: 

Lab 8-2 Step 5

#### **Expected behavior/Suggested Change**
Update needed listed above

#### **Additional context**

***Note**: To share feedback for the slides, please use the [MCT Courseware Forum](https://www.microsoft.com/learning/mct-courseware-forum.aspx).*

### PR DESCRIPTION
**Pull request by AngieRudduck**
Fixes #155, #156 , #158 , #159 , #162 , #163 

Changes proposed in this pull request:

- Updated Power BI service "Get Data" UI experience to use Upload instead.
- Updated minor file path issue in Dashboard lab.
- Updated Get Data instruction flow for Power BI dataset.

### CODE REVIEW

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md [hunk 1]
```diff
@@ -59,41 +59,37 @@ In this task, you'll set up the environment for the lab by signing in to Power B
 
 In this task, you'll set up the environment for the lab by creating a dataset.
 
-*Important: If you've already published the dataset in the **Create a Power BI Dashboard** lab, continue from the next task.*
+*Important: If you've already published the dataset, please move to the next task.*
```
Comment (Mihai-Ac) [inline Instructions/08-perform-data-analysis-in-power-bi-desktop.md:63]: This assumes that the student knows and completed one of the previous labs. I would just use
 _If you've already published the dataset in a previous lab, please move to the next task._

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md [hunk 2]
```diff
@@ -59,41 +59,37 @@ In this task, you'll set up the environment for the lab by signing in to Power B
...
-1. Save the file name as **Sales Exploration** and select **Save**.
-
-1. Next, we repeat the steps to **Get Data** from lab 1. From the **Home** ribbon tab > **Data** group, select **Power BI Datasets**.
+    *Tip: By default, the Getting Started dialog box opens in front of Power BI Desktop. You can choose to sign-in, and then close the pop-up.*
 
-	![Picture 15](Linked_image_Files/10-perform-data-analysis-in-power-bi-desktop_image12.png)
+1. In the Home ribbon, select **Get Data > Power BI datasets**.
 
-1. In the **Select a Dataset to Create a Report** window, select the **Sales Analysis** dataset, and then **Create**.
+1. In the **Select a Dataset to Create a Report** window, select the **Sales Analysis** dataset in **My Workspace**, and then **Connect** or double-click to load the dataset.
```
Comment (Mihai-Ac) [inline Instructions/08-perform-data-analysis-in-power-bi-desktop.md]: After selecting the **Get Data > Power BI datasets**  the window that popped up is not **Select a Dataset to Create a Report** , but **Data hub**, as displayed here. I suggesting changing that name to **Data hub**
image

File: Instructions/09-create-power-bi-dashboard.md [hunk 3]
```diff
@@ -87,17 +87,17 @@ In this task, you'll set up the environment for the lab by opening the starter r
 
 In this task, you'll set up the environment for the lab by creating a dataset.
 
-*Important: If you're continuing from the previous lab in the same VM, skip the next task.*
+*Important: If you've already published the dataset, please move to the next task.*
```
Comment (Mihai-Ac) [inline Instructions/09-create-power-bi-dashboard.md]: Just like in Lab 8, 
This assumes that the student knows and completed one of the previous labs. I would just use
If you've already published the dataset in a previous lab, please move to the next task.

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md [hunk 4]
```diff
@@ -59,41 +59,37 @@ In this task, you'll set up the environment for the lab by signing in to Power B
 
 In this task, you'll set up the environment for the lab by creating a dataset.
 
-*Important: If you've already published the dataset in the **Create a Power BI Dashboard** lab, continue from the next task.*
+*Important: If you've already published the dataset, please move to the next task.*
```
Comment (Mihai-Ac) [inline Instructions/08-perform-data-analysis-in-power-bi-desktop.md]: This assumes that the student knows and completed one of the previous labs. I would just use
If you've already published the dataset in a previous lab, please move to the next task.

**Review by Mihai-Ac — COMMENTED** [review]
All good with the exceptions I commented on. Please re-check those, I don't know if they are valid  or not.

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md (hunk 4 above)
Comment (shannonlindsay) [reply Instructions/08-perform-data-analysis-in-power-bi-desktop.md]: Agreed with Mihai! :)

**Review by shannonlindsay — COMMENTED** [review]
Looks great! I'd incorporate Mihai's suggestions and merge :)

File: Instructions/08-perform-data-analysis-in-power-bi-desktop.md (hunk 4 above)
Comment (AngieRudduck) [reply Instructions/08-perform-data-analysis-in-power-bi-desktop.md]: Suggestions implemented, thank you!

**Comment by VitalyMCT** [comment]
Thanks @AngieRudduck and @shannonlindsay. Have a great weekend.

**Review by Mihai-Ac — COMMENTED** [review]
Reviewed, all good from my side

**Pull request merged**